packages/bp/scripts/make_schemas.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs'
+import glob from 'glob'
+import mkdirp from 'mkdirp'
+import os from 'os'
+import path from 'path'
+import { getProgramFromFiles, generateSchema } from 'typescript-json-schema'
+
+const settings = {
+  required: true,
+  ignoreErrors: true
+}
+
+const bpDir = path.resolve(__dirname, '../')
+const files = glob.sync(path.join(bpDir, 'src/core/config/*.ts'))
+const program = getProgramFromFiles(files)
+
+const writeSchema = (typeName: string, jsonFile: string) => {
+  const definition = generateSchema(program, typeName, settings)
+  const json = JSON.stringify(definition, null, 2) + os.EOL + os.EOL
```

Comment:
Why add **two** end of line?

Reply:
Relic of the past. I don't remember why :/ That's from the beginning of BP, when we just merged PR without explaining the reason behind. Maybe formatting?